### ISSUE TITLE
syntax_error, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1711]
both method calls should determine the same void types

#1709

```abap
DATA: lr_typedescr TYPE REF TO cl_abap_typedescr.

CALL METHOD cl_abap_typedescr=>describe_by_name
  EXPORTING
    p_name = 'NAME'
  RECEIVING
    p_descr_ref = lr_typedescr
  EXCEPTIONS
    type_not_found = 1
    OTHERS = 2.

cl_abap_typedescr=>describe_by_name(
  EXPORTING
    p_name         = 'NAME'
  RECEIVING
    p_descr_ref    = lr_typedescr
  EXCEPTIONS
    type_not_found = 1
    OTHERS         = 2 ).
```

![image](https://user-images.githubusercontent.com/5888506/105817463-efb0e900-5fb5-11eb-9d65-72a9b3e912d5.png)
